README.md

```diff
@@ -30,16 +62,24 @@ import("foo.json", { type: "json" })
 <script src="foo.wasm" type="webassembly"></script>
 ```
 
+While the browser can check for WebAssembly's magic number. We would like to offer a way for the developer to assert the type of their ressources.
+
 ### WebAssembly
 
-Custom section: per import specifier, what options are needed
+Introduce a new custom section (named `importattributes`) that will annotate with attributes each imported module (based on the import section).
+
+Web WebAssembly embedding should check the attributes. 
```

Comment:
This phrasing is a bit turned around. I'd imagine these attributes would be passed to HTML, which would be the one who does the checking.

Reply:
Removed this line, it might be clear already.